### PR TITLE
Remove sass from few components

### PR DESCRIPTION
**Pull request by reginapramesti**
### What this PR does

Some work done on removing sass a while ago but forgot to push 😬 not related to any specific issue

### Checklist

-   [ ] There are unit tests to verify my changes are correct or unit tests aren't applicable (if so, write quick reason why unit tests don't exist)
-   [ ] I've updated CHANGES.md with what I changed.

### CODE REVIEW

File: lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ class HelpPanel extends React.Component {
 
   render() {
     const { t } = this.props;
-    // const isVisible = this.props.viewState.showHelpMenu;
+    console.log(this.props.theme);
```
Comment (soyarsauce) [inline lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx]: stray log

**Review by soyarsauce — COMMENTED** [review]
Related: https://github.com/TerriaJS/terriajs/issues/4545

FYI @reginapramesti I don't see any sass removed here 😜 (but lots of removed inlined-css-prop styles which is always always good) - and I get that the draft state is because the sass classname replacements are yet to be finalised, or is there even more to that?

**Comment by soyarsauce** [comment]
image

& re https://github.com/TerriaJS/terriajs/pull/4616#discussion_r466067364, another way to inspect the theme is on the `<ThemeProvider />` provider using the react inspector (but don't let that stop you from doing `console.log`s in development, it's simply another alternative)

**Comment by reginapramesti** [comment]
Oh nice, didn't know about inspecting the theme, that's really handy! Also whoops my bad I didn't realise there was an issue for removing sass!

I guess "few components" was misleading 😆 I removed the sass in a few places in HelpPanel, mainly from what used to be this big `className` thing:
```
    const className = classNames(
      {
        [Styles.helpPanel]: true,
        [Styles.isVisible]: isVisible && !isExpanded,
        [Styles.isHidden]: !isVisible,
        [Styles.helpPanelShifted]: isVisible && isExpanded
      },
```
into a Box component. I just dropped it here last night without checking whether or not it's ready for review hence the draft status (this was sitting in my personal PC for a month now without being pushed, so just wanted to get it out of there 😆) I think it's about everything I had

**Comment by soyarsauce** [comment]
@reginapramesti could you just do a quick changelog entry & we'll merge

**Review by soyarsauce — APPROVED** [review]

**Comment by soyarsauce** [comment]
Rerunning CI due to saucelabs timeout

**Comment by soyarsauce** [comment]
Actually 3f1945 is only a changelog change

**Pull request merged**